core/storage/base_model/gae_models.py

```diff
@@ -1224,18 +1228,17 @@ def delete_multi( # type: ignore[override]
                 snapshot_content_models.append(
                     model.SNAPSHOT_CONTENT_CLASS.create(snapshot_id, snapshot))
 
-            # We need the cast here in order to convey to MyPy that all these
-            # lists can be merged together for the purpose of putting them into
-            # the datastore.
-            entities = (
-                cast(List[BaseModel], snapshot_metadata_models) +
-                cast(List[BaseModel], snapshot_content_models) +
-                cast(List[BaseModel], versioned_models)
-            )
+            entities: List[BaseModel] = []
+            for snapshot_model in snapshot_metadata_models:
+                entities.append(snapshot_model)
+            for content_model in snapshot_content_models:
+                entities.append(content_model)
+            for versioned_model in versioned_models:
+                entities.append(versioned_model)
             cls.update_timestamps_multi(entities)
             BaseModel.put_multi_transactional(entities)
 
-    # We use type Any for *args and **kwargs to denote compatibility with the
+    # Here we use type Any because to denote the compatibility with the
```

Comment:
Drop "because"

Reply:
No, we cannot because we are checking for Any type comment with the help of `Here we use type Any because`.

Reply:
Then update the sentence, because currently it isn't grammatically correct ("because to ...")

Reply:
Updated PTAL!